tutorials/editor/upgrading_to_godot_4.rst

```diff
@@ -0,0 +1,544 @@
+  ``Time.get_time_dict_from_system()["second"]``.
+- A :ref:`class_StreamPeerTCP` must have ``poll()`` called on it to update its
+  state, instead of relying on ``get_status()`` automatically polling:
+  `GH-59582 <https://github.com/godotengine/godot/pull/59582>`__
+- ``is_connected_to_host()`` was removed from StreamPeerTCP and PacketPeerUDP as
+  per `GH-59582 <https://github.com/godotengine/godot/pull/59582>`__.
+  ``get_status()`` can be used in StreamPeerTCP instead.
+  ``is_socket_connected()`` can be used in PacketPeerUDP instead.
+
+**Removed or replaced nodes/resources**
+
+This lists all nodes that were replaced by another node requiring different
+configuration. The setup must be done from scratch again, as the project
+converter doesn't support updating existing setups:
+
++---------------------+-----------------------+----------------------------------------------------------------------+
+| Removed node        | Closest approximation | Comment                                                              |
++=====================+=======================+======================================================================+
+| AnimationTreePlayer | AnimationTree         | AnimationTreePlayer was deprecated since Godot 3.1.                  |
++---------------------+-----------------------+----------------------------------------------------------------------+
+| BakedLightmap       | LightmapGI            | See :ref:`doc_baked_lightmaps`.                                      |
++---------------------+-----------------------+                                                                      |
+| BakedLightmapData   | LightmapGIData        |                                                                      |
++---------------------+-----------------------+----------------------------------------------------------------------+
+| BitmapFont          | FontFile              | See :ref:`doc_gui_using_fonts`.                                      |
++---------------------+-----------------------+                                                                      |
+| DynamicFont         | FontFile              |                                                                      |
++---------------------+-----------------------+                                                                      |
+| DynamicFontData     | FontFile              |                                                                      |
++---------------------+-----------------------+----------------------------------------------------------------------+
+| Navigation3D        | Node3D                | Replaced by other Navigation nodes.                                  |
++---------------------+-----------------------+----------------------------------------------------------------------+
+| Navigation2D        | Node2D                | Replaced by other Navigation nodes.                                  |
```

Comment:
I'd add a link to the [2D ](https://docs.godotengine.org/en/latest/tutorials/navigation/navigation_introduction_2d.html)and [3D ](https://docs.godotengine.org/en/latest/tutorials/navigation/navigation_introduction_3d.html)navigation overview pages here.